src/main/java/uk/gov/hmcts/darts/transcriptions/exception/TranscriptionApiError.java

```diff
@@ -52,6 +52,11 @@ public enum TranscriptionApiError implements DartsApiError {
         "108",
         HttpStatus.BAD_REQUEST,
         "Failed to attach transcript"
+    ),
+    INTERNAL_SERVER_ERROR(
```

Comment:
Do we need to add another error - can we just use the one we have got already?
i.e. `TRANSCRIPTION_NOT_FOUND`

Reply:
Error 500 has now been removed from DMP-1329